routing/pathfind.go

```diff
@@ -1223,3 +1257,12 @@ func lastHopPayloadSize(r *RestrictParams, finalHtlcExpiry int32,
 	// The final hop does not have a short chanID set.
 	return finalHop.PayloadSize(0)
 }
+
+func safeAdd(x, y uint64) uint64 {
```

Comment:
Should we make this method take and return `lnwire.MilliSatoshi` instead? Then the call itself is a bit slimmer. Also, would nice to add a Godoc comment. And maybe "overflowSafeAdd" would be more fitting?